README.md

```diff
@@ -37,13 +39,30 @@ source venv/bin/activate
 pip install -r requirements.txt
 cp app/config.py.example app/config.py
 # Edit the variables' values.
-uvicorn app.main:app --reload
+# Rendering JWT_KEY:
+python -c "import secrets; from pathlib import Path; f = Path('app/config.py'); f.write_text(f.read_text().replace('JWT_KEY_PLACEHOLDER', secrets.token_hex(32), 1));"
 ```
 ### Running tests
 ```shell
 python -m pytest --cov-report term-missing --cov=app tests
 ```
 
+### Generating  personal JWT Secret Key
```

Comment:
Great job editing the `README.md` file. How about removing the two following paragraphs? It isn't a developer guide, just an initial readme to set the project up :)